docs/faq/subgraphAPI.md

```diff
@@ -0,0 +1,99 @@
+## How to Integrate Backend Libraries to MXNet with the Subgraph API
+
+The subgraph API has been proposed and implemented as the default mechanism for integrating backend libraries to MXNet. The subgraph API is a very flexible interface. Although it was proposed as an integration mechanism, it has been used as a tool for manipulating NNVM graphs for graph-level optimizations, such as operator fusion.
```

Comment:
As an entity or title, shouldn't it be "Subgraph API"? Then when you talk about a subgraph, it is lower case.